test/plugins/builds.test.js

```diff
@@ -4579,39 +4579,46 @@ describe('build plugin test', () => {
             };
         });
 
-        it('returns 200 when updating the code/endTime when the step model exists', () =>
-            server.inject(options).then(reply => {
+        it('returns 200 when updating the code/endTime when the step model exists', () => {
+            testStep.code = 0;
+            testStep.endTime = '2038-01-19T03:15:08.532Z';
+
+            return server.inject(options).then(reply => {
                 assert.equal(reply.statusCode, 200);
-                assert.deepProperty(reply.result, 'name', 'test');
-                assert.deepProperty(reply.result, 'code', 0);
-                assert.deepProperty(reply.result, 'endTime', options.payload.endTime);
-            }));
+                assert.nestedPropertyVal(reply.result, 'name', 'install');
```

Comment:
Why is the name changing from test to install

Reply:
because it was the the output of the mock, previous test never worked